File: samples/snippets/visualbasic/VS_Snippets_WindowsServer/wminet_SelectQuery_QueryString/vb/SelectQuery_QueryString.vb
'<Snippet1>
Imports System
Imports System.Management


Public Class Sample
    Public Overloads Shared Function _
        Main(ByVal args() As String) As Integer

        Dim s As New SelectQuery
        s.QueryString = "SELECT * FROM Win32_Process"

        Dim searcher As ManagementObjectSearcher
        searcher = New ManagementObjectSearcher(s)

        For Each o As ManagementObject In searcher.Get()
            'show the class
            Console.WriteLine(o.ToString())
        Next


    End Function 'Main
End Class 'Sample
'</Snippet1>

File: samples/snippets/csharp/VS_Snippets_WindowsServer/wminet_SelectQuery_QueryString/cs/SelectQuery_QueryString.cs
//<Snippet1>
using System;
using System.Management;

public class Sample 
{
    public static void Main(string[] args) 
    {
        SelectQuery s = new SelectQuery(); 
        s.QueryString = "SELECT * FROM Win32_Process";
      
        ManagementObjectSearcher searcher = 
            new ManagementObjectSearcher(
            s);

        foreach (ManagementObject o in searcher.Get()) 
        {
            // show the class
            Console.WriteLine(o.ToString());
        }
    }
}
//</Snippet1>